go.mod

```diff
@@ -5,15 +5,15 @@ go 1.20
 require (
 	github.com/avast/retry-go v3.0.0+incompatible
 	github.com/cortexproject/cortex v1.2.1-0.20200803161316-7014ff11ed70
-	github.com/ethereum/go-ethereum v1.10.26
+	github.com/ethereum/go-ethereum v1.11.2
```

Comment:
This is a higher version of go-ethereum than is being used in core. Could this cause issues?